kbase-extension/static/kbase/js/common/monoBus.js

```diff
@@ -42,6 +42,15 @@ define(['uuid', 'bluebird', './lang', './unodep'], (Uuid, Promise, lang, utils)
             }
         }
 
+        /**
+         * Clears all variables and the timer, if it exists
+         */
+        function destroy() {
+            if (timer) {
+                clearTimeout(timer);
```

Comment:
There's a "timer" that just waits for the current tick to finish. It's really a `setTimeout(()=>{}, 0)` function. I figured this would be prudent, just in case.

I guess it could be an edge case where that timeout fires right at the end of a test, then sends messages off to things that have been reaped. I'm not sure how to test for that, though.